Gemfile.lock

```diff
@@ -397,7 +396,7 @@ GEM
       activemodel (>= 5.0)
       bindex (>= 0.4.0)
       railties (>= 5.0)
-    webdrivers (3.8.0)
```

Comment:
There's a couple of major version updates in here, are they safe?

Reply:
I tested the branch on integration and the app seems to be working fine, so I think it is all safe. In particular `webdrivers` is only used to run the tests and they're still passing fine.